### PR TITLE
Adding git_push flow

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

File: content/io/cloudslang/git/git_push.sl [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+#
+####################################################
+# This flow performs a git push command to send a branch to a remote git repository
+#
+#   Inputs:
+#       - host - hostname or IP address
+#       - port - optional - port number for running the command
+#       - username - username to connect as
+#       - password - password of user
+#       - sudo_user - true or false, whether the command should execute using sudo
```
Comment (Bonczidai) [inline content/io/cloudslang/git/git_push.sl]: missing ` - Default: false` in the description

**Comment by Bonczidai** [comment]
good so far - need to add test :sunglasses:

**Comment by lirantal** [comment]
@Bonczidai updated - I think we also need to update the other git flows for that default  for sudo_user field.

What kind of test can I add? pushing to a repository requires permissions and I'm not sure how it would be easy to add test but I'm all ears if you have some kind of flow to suggest.

**Comment by Bonczidai** [comment]
I see here two possibilities:
- we could set up a Docker container with Git server on it - this could be used for all the Git flows
- add local test with `git-local` test suite

We can go with the first one which is a bit more complex or go with the second one and do the automation later - @meirwah WDYT on this?

**Comment by lirantal** [comment]
@Bonczidai both options sound somewhat complex to me so I would love to get your help on this if we want to go for any of them.

**Comment by Bonczidai** [comment]
sure - I'll be happy to guide you once we decide which option to go with

**Comment by meirwah** [comment]
let's start with "git-local" , sanity test, and add later complex (using docker) tests...

**Comment by lirantal** [comment]
I don't know what's "git-local", so whose taking this further?

**Comment by meirwah** [comment]
@orius123  can work on adding a sanity test...

**Pull request closed without merging**